style.css

```diff
@@ -169,4 +160,283 @@
     order: 3;
     flex-grow: 0;
   }
+
+  /* from here */
+  .snapshot1 {
+    margin-top: 30px;
+    height: 570px;
+    width: 375px;
+    border-radius: 16px;
+    padding: 16px;
+    background: white;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 12px;
+  }
+
+  body {
+    background: #e5e5e5;
+  }
+
+  .head-one {
+    /* Project Title */
+    width: 295px;
+    height: 20px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    margin-top: -20px;
+
+    /* Neutrals/N800 */
+    color: #172b4d;
+  }
+
+  .doc {
+    width: 295px;
+    height: 76px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+
+    /* Neutrals/N600 */
+    color: #344563;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  ul li {
+    text-decoration: none;
+  }
+
+  .grid-container {
+    margin-top: 30px;
+    display: grid;
+    grid-template-columns: auto auto auto;
+    height: 32px;
+    min-width: 310px;
+    align-items: center;
+  }
+
+  .grid-item {
+    display: flex;
+    justify-content: space-between;
+    font-size: 20px;
+    padding: 5px;
+    order: 1;
+  }
+
+  .counter {
+    height: 10px;
+    margin-top: 8px;
+    margin-right: 5px;
+  }
+
+  .tags {
+    height: 24px;
+    width: 224px;
+    top: 200px;
+  }
+
+  .canop {
+    color: #020303;
+  }
+
+  #grey-works {
+    color: #7a869a;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    justify-content: space-between;
+    column-gap: 10px;
+  }
+
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    font-size: 14px;
+  }
```

Comment:
- [ ] Good job so far 👏. Please try to center the text inside each box in this part of project card. And this applies to all the project cards 🏃 Please take a look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/192117825-d2dc5bcd-b646-42fc-bb76-a1071238a0e6.png)
**Figma design**
![image](https://user-images.githubusercontent.com/90222110/192117840-978f8203-d80a-4132-823f-2138d3656da4.png)